docs/releases/lifecycles/index.md

```diff
@@ -57,3 +67,74 @@ If you want to be able to deploy to any environment at any time, then simply cre
+
+In the next section, we look at configuring the default lifecycle to add your own phases.
+
+### Adding a phase to the default lifecyle
+
+You can define your own phases for the default lifecycle. This helps to prevent having too many phases being added automatically. To add a new phase, in the default lifecycle, Click **ADD PHASE**. Here, we are creating a phase named **Development** and adding the Dev environment to the phase:
+
+![Add Dev lifecycle phase](images/default-lifecycle-add-dev-phase.png "width=500")
+
+This phase has the default option to manually deploy to the environment set. The Required to progress and Retention policy are also set to the default values.
+
+:::hint
+Phase names usually match the environment it contains. While this is a good practice, it's not a rule.
+:::
+
+You can repeat this process to create extra phases. In this example, we are creating a phase for Testing, Staging, and Production.
+
+![Default lifecycle phases added](images/default-lifecycle-phases-added.png "width=500")
+
+This allows you to explicitly configure the default lifecycle for deploying your software.
+
+## Examples {#lifecycle-examples}
+
+In this section, we cover some lifecycle examples and their included phases.
+
+### Hotfix lifecycle
+
+A hotfix lifecycle is useful when you have a critical bug-fix that needs to be deployed quickly. In this scenario, lower environments such as Development and Testing are skipped. Deploying to both typically takes too much time.
+
+It's recommended to follow good deployment practices and validate any changes before pushing to production. To match this, a hotfix lifecycle usually has just two phases, Staging and Production. Software with the bug fix is validated in Staging, and then promited to Production. However your lifecycle may be different to reflect how you decide to handle hotfixes.
+
+![Hotfix lifecycle](images/hotfix-lifecycle.png "width=500")
+
+### Maintenance lifecycle
+
+A Maintenance lifecycle can be used for projects that run maintenance tasks such as backups or software upgrades. This lifecycle can be used for any tasks that you want to run regularly with the same benefits that Octopus provides for your application deployments.
+
+It typically consists of just one phase and one environment, also called Maintenance. You can include this environment in all deployment targets you want to run these tasks against. You can also split them up into the Development, Testing, Staging, and Production environments if you want to run the tasks for targets in those environments at different times.
+
+![Maintenance lifecycle](images/maintenance-lifecycle.png "width=500")
+
+## Recommendations {#lifecycle-recommendations}
+
+When configuring your lifecycles, here are some tips to consider:
+
+- Update the default lifecycle to define the phases you need. This makes configuring and maintaining your Octopus Server easier.
```

Comment:
I need to turn my script for this into an api example

Reply:
That'd be good, we could link to it in this section once its ready?